extensions/typescript-language-features/src/utils/configuration.ts

```diff
@@ -80,6 +81,7 @@ export class TypeScriptServiceConfiguration {
 		this.useSeparateSyntaxServer = TypeScriptServiceConfiguration.readUseSeparateSyntaxServer(configuration);
 		this.enableProjectDiagnostics = TypeScriptServiceConfiguration.readEnableProjectDiagnostics(configuration);
 		this.maxTsServerMemory = TypeScriptServiceConfiguration.readMaxTsServerMemory(configuration);
+		this.enablePromptUseWorkspaceTsdk = TypeScriptServiceConfiguration.readEnablePromptUseWorkspaceTsdk(configuration);
```

Comment:
If you add a setting here, make sure to also add a check for it in the `isEqual` function below